creating/export-blender-to-aframe/index.md

```diff
@@ -18,3 +20,154 @@ Using [A-Frame](https://aframe.io/), we will show you how to export a Blender mo
+
+## Unzip/extract the blend file
+Right click on the .zip file and select Extract All. Save the extracted contents to a directory you can find easily.
+
+![Extract Zip Contents](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExtractAll.png)
+
+## Open the blend file with Blender
+Take a quick look at the Blender user interface. It has a high learning curve, but can be a powerful tool for creating 3D graphics. For now, we’ll just use it as an exporter for the CV1 model.
+
+## Export the model: File → Export → Wavefront (.obj)
+Name and save the .obj file (saved by default in same directory with the blend file)
+
+![Export .obj from Blender](https://github.com/WebVRRocks/webvrrocks/tree/master/creating/export-blender-to-aframe/img/ExportOBJ.png)
+
+## Copy the .obj and .mtl files into the directory with your index.html file
+Go back to the blend file in Blender and export as Collada (.dae)
+In the directory with your blend file, you should now have a .dae file, plus a .obj file and a .mtl file that goes with it. Note that there are a number of .jpg image files that came with the model. These are textures for the .obj: they are important! Copy-paste the .dae, .obj .mtl and all of the .jpg files into the directory with your index.html file.  
+
+
+# PART 3: WEB PAGE AND LOCAL SERVER
+---
+
+## Open your index.html file in your favorite text editor
+In the boilerplate, note that I’ve provided attribution to the model’s creator in a comment in lines 1-3. If you are new to A-Frame, there are some helpful comments in the code, as well.
+Because the 3D model files are in the same directory with your index.html, the file paths to them simply consist of their names. Go to line 26, which is within <a-assets>. This is  where we’ll initially load the assets for the scene. In src=”cv1_FF.obj” change cv1_FF.obj to match the name you gave your .obj file. Do the same for the .mtl and the .dae asset items in lines 27 and 28.
+Note that each model is given an ID, which is used to instantiate them into the scene as entities in lines 39 and 40. Using an asset system like this has the advantage of pre-loading your models, and you can then instantiate them as many times as you want without sacrificing as much performance.
+
+## Start up your local http server 
+Open the command prompt. Type:
+
+'''
+http-server [full\path\to\directory containing index.html] -o
+'''
+
+The -o option will open a default browser with your index.html webpage. The default port it uses is 8080; there’s a option to change this, if you want. https://www.npmjs.com/package/http-server
+
+## Can you see the models in the webpage?
+The Collada .dae will be on your left, and the Wavefront .obj on your left. Click-and-drag to rotate the camera and look around. You can also move with the traditional W-A-S-D keys. If you can’t see the models, check the file paths in <a-assets> and also the IDs in the obj-model and collada-model entities.
```

Comment:
I'm confused by this: "entities -> A-Frame components (HTML attributes) on the entities (i.e., the <a-entity> HTML elements)"

Do you mean that I should refer to the entities that contain the models as "A-Frame components"?